standard/count.lua

```diff
@@ -0,0 +1,175 @@
+local Class = require('Module:Class')
```

Comment:
suggest to move it to `./standard/lpdb` folder since it is for lpdb count queries